### DISCUSSION TITLE
Person who started the meeting always goes first

### DISCUSSION THREAD

**Discussion by avivapinchas** [ParabolInc/parabol#4772] [Feedback]
Since we implemented the change where the only folks who show up in a meeting are those that have actually showed up (vs the whole team), the person who started the meeting is always first in the lineup, which means you answer the icebreaker first. 

As a team lead, I'm usually the one starting the meeting, and I don't want to always answer the icebreaker first. I'd like the option for that lineup to be more random, maybe by hitting a 'randomize!' button or maybe something else. 

@ackernaut @jordanh what says ye?

**Comment by ackernaut** [comment]
I think that the first stage of the Icebreaker phase is a moment when everybody present takes a few minutes to type out their answer on a card (like the Reflect phase). When the facilitator advances (‘Is everybody ready??’) stages are added for each participant in random order. Now their stage will have both their picture and their typed response. They can still take a moment to expand on it over video.

Later we add options for input beyond typing: add a Gif, add a drawing, make an async video

Perhaps questions optionally have a response type: question 123 allows drawings only, question 231 is Gifs only

Later we allow for Icebreaker UGC

Later we build a teammate trivia game that the team lead can set up (cadence) for their team. The folks get prompted in Slack to go play the game.

I can think of a couple of formats:

How did Steve answer this question? ‘What’s your favorite food?’ [ 4 answers from 4 teammates, 1 is Steve’s answer]
Who gave this answer? ‘What’s your favorite food?’ ‘Anchovies’ [ 4 teammate names, with Steve as 1 possible answer ]

**Reply by jordanh** [reply]
Yeah, this is better. This would be a _great_ design BP (or first project), I think

**Comment by jordanh** [comment]
Hi @nickoferrall

**Comment by jordanh** [comment]
@ParabolInc/parabol-product this is a team

**Comment by enriquesanchez** [comment]
Created implementation issue #4910 to address a quick solution to randomize order of participants.
